docs/resources/Channel.md

```diff
@@ -388,30 +388,28 @@ See [permissions](#DOCS_TOPICS_PERMISSIONS/permissions) for more information abo
 
 Receiving:
 
-| Field     | Type      | Description               |
-| --------- | --------- | ------------------------- |
-| id        | snowflake | role or user id           |
-| type      | string    | either "role" or "member" |
-| allow     | integer   | legacy permission bit set |
-| allow_new | string    | permission bit set        |
-| deny      | integer   | legacy permission bit set |
-| deny_new  | string    | permission bit set        |
+| Field | Type      | Description                   |
+|-------|-----------|-------------------------------|
+| id    | snowflake | role or user id               |
+| type  | int       | either 0 (role) or 1 (member) |
+| allow | string    | permission bit set     |
+| deny  | string    | permission bit set     |
 
 Sending:
 
-| Field | Type              | Description               |
-| ----- | ----------------- | ------------------------- |
-| id    | snowflake         | role or user id           |
-| type  | string            | either "role" or "member" |
-| allow | integer or string | permission bit set        |
-| deny  | integer or string | permission bit set        |
+| Field | Type      | Description                   |
+|-------|-----------|-------------------------------|
+| id    | snowflake | role or user id               |
+| type  | int       | either 0 (role) or 1 (member) |
+| allow | string    | permission bit set            |
+| deny  | string    | permission bit set            |
```

Comment:
If permissions can't be sent as ints now, then these tables shouldn't be duplicated anymore, as they are the same now (I only separated these when _new fields were added)

Reply:
tpcsld says permissions can still be sent as ints or strings, but this PR changes it to only say they can be sent as strings; Is sending it as ints considered deprecated or was this mistakenly changed?

Reply:
@msciotti we might want to talk about this :'D

* For the time being, the API is definitely going to support both JSON int and strings, since we just cast it to a number internally.
* Nevertheless, our official clients will only ever send strings.
* It is entirely up to you whether or not we want to publicly deprecate ints here.

My 2 cents is maybe keep ints here and deprecate in the future if we have to? /shrug Or say that we prefer strings here?